pkg/inventory/web/handler.go

```diff
@@ -150,62 +150,55 @@ func (r *Event) String() string {
 
 //
 // Watch (event) writer.
+// The writer is model event handler. Each event
+// is send (forwarded) to the watch client.  This
+// provides the bridge between the model and web layer.
 type WatchWriter struct {
 	// negotiated web socket.
 	webSocket *websocket.Conn
-	// model watch.
-	watch *model.Watch
```

Comment:
We don't need/want to store a reference to the watch.  This was only needed so that the writer can end the watch when it detects the peer (client) has closed the connection.  This is unwanted because it creates a bi-directional references watch.Handler = WatchWriter -> watch which prevents garbage collection.